tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+/*
+ * Angular Material Data Table
+ * https://github.com/daniel-nagy/md-data-table
+ * @license MIT
+ * v0.10.9
+ */
+(function (window, angular, undefined) {
+'use strict';
```

Comment:
Codacy has a fix for the issue: ['use strict' is unnecessary inside of modules.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion

```